bootstrap/components/sources/kafka-source-topology-component.json

```diff
@@ -35,13 +35,23 @@
         "hint": "schema,override"
       },
       {
-        "uiName": "Reader schema version",
+        "uiName": "Reader Schema Branch",
+        "fieldName": "schemaBranch",
+        "isOptional": false,
+        "tooltip": "Schema branch to be used fetching for reader schema versions. Default branch is MASTER",
+        "type": "enumstring",
+        "options": [],
+        "hint": "dependsOn-topic,schemaBranch",
+        "defaultValue": "MASTER"
```

Comment:
@satishd same thing over here, failing the source to be registered during bootstrap.